.github/workflows/check.yml

```diff
@@ -98,8 +98,8 @@ jobs:
         run: python -m pip install uv
       - name: Uninstall pip
         run: python -m pip uninstall pip -y
-      - name: Active uv for global env
-        run: echo "VIRTUAL_ENV=${Python_ROOT_DIR}" >> $GITHUB_ENV
+      - name: Allow uv to use the global Python
+        run: echo "UV_SYSTEM_PYTHON=1" >> $GITHUB_ENV
```

Comment:
You could also just use the `--system` flag here, not sure if you need it active for the later `tox` invocation though?